lib/hexpm_web/templates/package/show.html.eex

```diff
@@ -120,50 +125,49 @@ tools = Keyword.take(tools, compatible_tools)
+    </ul>
 
-    <div class="row">
-      <div class="col-md-6">
-        <h3>Versions (<%= Enum.count(@releases) %>)</h3>
-        <ul id="versions" class="version-dependency-list">
-          <% show_latest_versions = 5 %>
-          <%= render "versions.html", [package: @package, releases: Enum.take(@releases, show_latest_versions)] %>
-        </ul>
-
-        <%= if length(@releases) > show_latest_versions do %>
-          <p class="show-versions">
-            <a href="<%= path_for_releases(@package) %>">Show All Versions</a>
-          </p>
+    <%= if length(@releases) > show_latest_versions do %>
+      <p class="show-versions">
+        <a href="<%= path_for_releases(@package) %>">Show All Versions</a>
+      </p>
+    <% end %>
+  </div>
+  <div class="col-md-6 no-padding">
+    <h3>Dependencies <span class="attr-count">(<%= if @current_release, do: Enum.count(@current_release.requirements), else: 0 %>)</span></h3>
+    <ul class="version-dependency-list">
+      <%= if @current_release do %>
+        <%= for %{repository: repository, name: name, requirement: req, app: app, optional: optional} <- @current_release.requirements do %>
+        <li>
+          <a href="<%= path_for_package(repository, name) %>"><%= package_name(repository, name) %></a> <%= req %>
+          <%= if app != name do %>(app: <%= app %>)<% end %>
+          <%= if optional do %><span class="attr-optional">(optional)</span><% end %>
+        </li>
         <% end %>
-      </div>
-      <div class="col-md-6">
-        <h3>Dependencies (<%= if @current_release, do: Enum.count(@current_release.requirements), else: 0 %>)</h3>
-        <ul class="version-dependency-list">
-          <%= if @current_release do %>
-            <%= for %{repository: repository, name: name, requirement: req, app: app, optional: optional} <- @current_release.requirements do %>
-            <li>
-              <a href="<%= path_for_package(repository, name) %>"><%= package_name(repository, name) %></a> <%= req %>
-              <%= if app != name do %>(app: <%= app %>)<% end %>
-              <%= if optional do %>(optional)<% end %>
-            </li>
-            <% end %>
-          <% end %>
-        </ul>
-      </div>
-    </div>
+      <% end %>
+    </ul>
+  </div>
+  </div>
 
-    <div class="row">
-      <div class="col-md-12">
-        <h3>Audit Logs</h3>
-        <ul class="audit-logs-list">
-          <%= for audit_log <- @audit_logs do %>
-            <li>
-              <%= pretty_date(audit_log.inserted_at) %> <%= humanize_audit_log_info(audit_log) %>
-            </li>
-          <% end %>
-        </ul>
-      </div>
-    </div>
+  <div class="col-md-11 no-padding">
+    <h3>Recent Activity</h3>
+    <ul class="audit-logs-list">
+      <%= for audit_log <- @audit_logs do %>
+        <li>
+          <%= pretty_datetime(audit_log.inserted_at, :short) %> <%= humanize_audit_log_info(audit_log) %>
```

Comment:
should we keep the name `pretty_date` given we're not displaying time? Otherwise, let's remove pretty_date as it became obsolete.